src/lib/modules/Items/backoffice/ItemListEntry.js

```diff
@@ -78,23 +79,57 @@ export class ItemListEntry extends Component {
             <Grid.Column computer={6} largeScreen={6}>
               <Item.Meta className="metadata-fields">
                 <Header as="h5">
-                  <LiteratureTitle
-                    title={item.metadata.document.title}
-                    truncateLines={1}
-                  />
+                  {checkIn ? (
```

Comment:
I think this change should be reverted, since it does not add so much in terms of the user experience, and it might be confusing to have this element in different places, depending on the location. I think we should keep things as much harmonised as possible